BookingsAPI/Bookings.API/Properties/launchsettings.json

```diff
@@ -3,7 +3,7 @@
   "profiles": {
     "BookingsApi": {
       "commandName": "Project",
-      "launchBrowser": true,
+      "launchBrowser": false,
```

Comment:
What will this do @rehmanab?

Reply:
It will not start the browser, this is an api, so no need to start the web page. @npoxon

Reply:
okay cool